conf/app.ini

```diff
@@ -40,67 +39,57 @@ redis.sessions.timeout: 604800
 # SQLAlchemy configuration -- See SQLAlchemy documentation
 sqlalchemy.url: postgresql://postgres@localhost/postgres
 
-
 [filter:proxy-prefix]
 use: egg:PasteDeploy#prefix
 
-
 [filter:raven]
 use: egg:raven#raven
 
-
 [server:main]
 use: egg:gunicorn
 host: 0.0.0.0
 port: 5000
 
-
 [loggers]
-keys = root, sentry, gunicorn.error, alembic
-
+keys = root, alembic, gunicorn.error, sentry
 
 [handlers]
 keys = console, sentry
 
-
 [formatters]
 keys = generic
 
-
 [logger_root]
+level = WARNING
```

Comment:
They're the same, but [`WARNING` is the canonical name](https://docs.python.org/2/library/logging.html#levels).